concent_api/core/payments/service.py

```diff
@@ -99,4 +99,21 @@ def call_on_confirmed_transaction(
     tx_hash: str,
     callback: Callable
 ) -> None:
-    backend.call_on_confirmed_transaction(tx_hash, callback)
+    backend.call_on_confirmed_transaction(
+        tx_hash,
+        callback,
+    )
+
+
+@_add_backend
+def validate_that_last_closure_time_is_older_than_oldest_payment(
```

Comment:
Same here.

Reply:
Same as above

Reply:
Each of this functions must also exists in each backends. As it doesn't exist in `sci_backend`, it can't exist here.

Reply:
removed, this function simply uses other 'backend' function and this function is mocked